src/utils/audio-manager.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+const playMock = jest.fn();
+Audio.prototype.play = async () => playMock();
+let getAudioManager;
+
+beforeEach(() => {
+  jest.resetModules();
+  getAudioManager = require('./audio-manager').default;
```

Comment:
Don't use `require` in your files. We use ES6 `import`/`export`.

You can import it once at the beginning of your file and it should work just the same (or actually even faster, because you won't import it multiple times).

Reply:
I've tried to do it this way, but then these tests affect each other. Example:

`test('Check if getAudioManager() has been called with no args then play() returns undefined ', async () => {
  const instanceOfAudioManager = getAudioManager();
  await instanceOfAudioManager.play();

  expect(playMock).not.toHaveBeenCalled();
});

test('Check if getAudioManager() has been called with the argument, then play() is called ', async () => {
  const instanceOfAudioManager = getAudioManager(
    'http://icepool.silvacast.com/COUNTRY108.mp3'
  );
  await instanceOfAudioManager.play();
  expect(playMock).toHaveBeenCalled();
});`

The second test pass, if the first one will skipped, even though I use jest.resetModules() before each test.